Formula/mysql-cluster.rb

```diff
@@ -38,10 +37,20 @@ class MysqlCluster < Formula
     cause "https://article.gmane.org/gmane.comp.db.mysql.cluster/2085"
   end
 
+  resource "boost" do
+    url "https://downloads.sourceforge.net/project/boost/boost/1.59.0/boost_1_59_0.tar.bz2"
+    sha256 "727a932322d94287b62abb1bd2d41723eec4356a7728909e38adb65ca25241ca"
+  end
+
   def install
     # Make sure the var/mysql-cluster directory exists
     (var/"mysql-cluster").mkpath
 
+    # dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
+    if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+      inreplace "configure.cmake", "(clock_gettime", "(everything_is_terrible"
```

Comment:
It felt appropriate 😉. I hadn't yet, no. To be honest if you went through my list of favourite bug reporting systems & how those bugs are handled MySQL stuff would be almost glued to the bottom 😕